app/src/main/kotlin/com/derek_s/hubble_gallery/detailspage/DetailsPresenter.java

```diff
@@ -31,25 +31,26 @@
 
 @Singleton
 public class DetailsPresenter {
-  public TileObject tileObject;
-  public DetailsObject detailsObject;
-  public String successfulSrc;
-  public int scrollYPos;
-  public int darkVibrantColor;
-  public int lightVibrantColor;
-  public int toolbarBgColorAlpha;
-  @Inject
-  Context context;
-  @Inject
-  Resources resources;
-  @Inject
-  FavoriteUtils favoriteUtils;
-  private String TAG = getClass().getSimpleName();
+  private static final String TAG = DetailsPresenter.class.getSimpleName();
```

Comment:
oh man you're a saint, this was so messy before

Reply:
Ha-ha! Saint =) I'll print it out and put in a frame =)